apps/ejabberd/priv/cassandra.cql

```diff
@@ -1,4 +1,77 @@
-DROP KEYSPACE IF EXISTS mongooseim;
 CREATE KEYSPACE mongooseim
          WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3};
 USE mongooseim;
+
+CREATE TABLE mam_message(
```

Comment:
No, it's not compatible. Even though schema is only slightly different, there are major differences in how messages are stored - there is message duplication in order to give full lookup functionality. At first I've tried to run this backend from 1.6.2, but most of the test wouldn't work and after investigating - they shouldn't.